coffee_roaster_market/apps/conftest.py

```diff
@@ -0,0 +1,18 @@
+import pytest
```

Comment:
I didn't know if it's good idea to place global conftest here, but in my opinion implementing a `client` fixture for each app or importing it from `apps.coffee.tests` would be weird. @kuchichan what do you think about that?

Reply:
I think it's fine.